packages/hash/api/src/index.ts

```diff
@@ -102,22 +99,7 @@ const main = async () => {
     10,
   );
 
-  await Promise.all([
-    waitOnResource(`tcp:${pgHost}:${pgPort}`, logger),
-    waitOnResource(`tcp:${redisHost}:${redisPort}`, logger),
-  ]);
-
-  // Connect to the database
-  const pgConfig = {
-    host: pgHost,
-    user: getRequiredEnv("HASH_PG_USER"),
-    password: getRequiredEnv("HASH_PG_PASSWORD"),
-    database: getRequiredEnv("HASH_PG_DATABASE"),
-    port: pgPort,
-    maxPoolSize: 10, // @todo: needs tuning
-  };
-  const db = new PostgresAdapter(pgConfig, logger, statsd);
-  shutdown.addCleanup("Postgres", async () => db.close());
+  await Promise.all([waitOnResource(`tcp:${redisHost}:${redisPort}`, logger)]);
```

Comment:
We don't need the Promise.all anymore right?

Reply:
Correct, addressed in https://github.com/hashintel/hash/pull/889/commits/0e1af9c2071db54d2b4c3fb6abe7ea62efc66f60